packages/visx-demo/src/components/Show.tsx

```diff
@@ -21,15 +21,15 @@ type ShowProps = {
   events?: boolean;
   margin?: MarginShape;
   description?: Component<{ width: number; height: number }>;
-  windowResizeDebounceTime?: number;
+  debounceTime?: number;
   packageJson?: PackageJson;
 };
 
 const padding = 40;
 
-export default withScreenSize<ShowProps & WithScreenSizeProvidedProps>(
+export default withScreenSize<ShowProps & WithSizeProvidedProps>(
   ({
-    screenWidth,
+    width,
```

Comment:
hey @susiwen8 thanks for the PR! I'll try to take a proper review pass either today or tomorrow but one thing I'd note is that we should try to avoid any API changes because those are breaking changes and will require us to release a `2.0` version.

If you'd like to improve/unify APIs, I think that's great 🎉  but I'd recommend supporting _both_ sets of props for now, and then we'd deprecate/officially remove the old ones when we want to do the `2.0` release.

Reply:
@williaster  Hi, I had no confident on this one. Will take a look.